Hw12/GameFileWork.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <tuple>
+#include <vector>
+#include <string>
+
+class GameFileWork
+{
+public:	
+	static GameFileWork &instance();
+
+	void makeImpossibleWordOfDay() const;
```

Comment:
static

Reply:
as it doesn't work with any properties of the class

Reply:
Ok